tools/styles/tokens-v2/system-theme-bridge.css

```
@@ -631,13 +631,17 @@
     --system-tag-size-large-clear-button-spacing-inline-end: var(--spectrum-component-edge-to-visual-200);
     --system-textfield-background-color: var(--spectrum-gray-25);
     --system-textfield-background-color-disabled: var(--spectrum-gray-25);
-    --system-textfield-border-color: var(--spectrum-gray-500);
-    --system-textfield-border-color-hover: var(--spectrum-gray-600);
+    --system-textfield-border-color-hover: var(--spectrum-gray-400);
     --system-textfield-border-color-focus: var(--spectrum-gray-800);
     --system-textfield-border-color-focus-hover: var(--spectrum-gray-900);
     --system-textfield-border-color-keyboard-focus: var(--spectrum-gray-800);
-    --system-textfield-border-color-disabled: var(--spectrum-gray-300);
+    --system-textfield-border-color-disabled: var(--spectrum-disabled-border-color);
     --system-textfield-border-width: var(--spectrum-border-width-100);
+    --system-textfield-border-color: var(--spectrum-gray-300);
+    --system-textfield-spacing-block-start: var(--spectrum-component-padding-vertical-75);
+    --system-textfield-spacing-block-start-small: var(--spectrum-component-padding-vertical-100);
+    --system-textfield-spacing-block-start-large: var(--spectrum-component-padding-vertical-200);
+    --system-textfield-spacing-block-start-extra-large: var(--spectrum-component-padding-vertical-300);
```

I'm not sure SWC has taken in the update with the `component-padding-vertical-*` tokens. SWC also doesn't quite yet support the new variable Adobe Clean font, and these would be the tokens used in conjunction with the new font. 

Would we be able to compromise by using the un-updated versions of this spacing? It's not exactly the same values, but these would correspond to the padding values for the old font.

```suggestion
    --system-textfield-spacing-block-start: var(--spectrum-component-top-to-text-100);
    --system-textfield-spacing-block-start-small: var(--spectrum-component-top-to-text-75);
    --system-textfield-spacing-block-start-large: var(--spectrum-component-top-to-text-200);
    --system-textfield-spacing-block-start-extra-large: var(--spectrum-component-top-to-text-300);
```